script/releases/v1.0.0-slashing/cleanup/script.go

```diff
@@ -0,0 +1,311 @@
+		fmt.Printf("Completing [%d/%d]...", i+1, len(results))
+		fmt.Printf("NOTE: this is expensive, and may take several minutes.")
+		completeCheckpointForEigenpod(ctx, results[i].Address, eth, chainId, coreBeaconClient, args.Sender)
+	}
+
+	return nil
+}
+
+func completeCheckpointForEigenpod(ctx context.Context, eigenpodAddress string, eth *ethclient.Client, chainId *big.Int, coreBeaconClient proofgen.BeaconClient, sender string) {
+	res, err := proofgen.GenerateCheckpointProof(ctx, eigenpodAddress, eth, chainId, coreBeaconClient, true)
+	panicOnError(fmt.Sprintf("failed to generate checkpoint proof for eigenpod:%s", eigenpodAddress), err)
+
+	txns, err := proofgen.SubmitCheckpointProof(ctx, sender, eigenpodAddress, chainId, res, eth, 80 /* ideal checkpoint proof batch size */, true /* noPrompt */, false /* noSend */, true /* verbose */)
+	panicOnError(fmt.Sprintf("failed to submit checkpoint proof for eigenpod:%s", eigenpodAddress), err)
+	if txns == nil {
+		panic("submitting checkpoint proof generated no transactions. this is a bug.")
+	}
+
+	for i, txn := range txns {
+		fmt.Printf("[%d/%d] %s\n", i+1, len(txns), txn.Hash())
+	}
+}
+
+// This is a simplified version of the queryAllEigenpodsOnNetwork function inline.
+// It uses the logic from the provided code snippet in the commands package.
+func queryAllEigenpodsOnNetwork(
+	ctx context.Context,
+	allValidators []ValidatorWithIndex,
+	eth *ethclient.Client,
+	eigenpodAbi, podManagerAbi *abi.ABI,
+	podManagerAddress string,
+	mc *multicall.MulticallClient,
+) ([]string, error) {
+	args := TQueryAllEigenpodsOnNetworkArgs{
+		Ctx:               ctx,
+		AllValidators:     allValidators,
+		Eth:               eth,
+		EigenpodAbi:       *eigenpodAbi,
+		PodManagerAbi:     *podManagerAbi,
+		PodManagerAddress: podManagerAddress,
+		Mc:                mc,
+	}
+	return internalQueryAllEigenpodsOnNetwork(args)
+}
+
+// internalQueryAllEigenpodsOnNetwork is lifted from the provided snippet.
+func internalQueryAllEigenpodsOnNetwork(args TQueryAllEigenpodsOnNetworkArgs) ([]string, error) {
+	// Filter out validators that are withdrawing to execution layer addresses
+	executionLayerWithdrawalCredentialValidators := lo.Filter(args.AllValidators, func(validator ValidatorWithIndex, i int) bool {
+		return validator.Validator.Validator.WithdrawalCredentials[0] == 1
+	})
+
+	interestingWithdrawalAddresses := lo.Keys(lo.Reduce(executionLayerWithdrawalCredentialValidators, func(accum map[string]int, next ValidatorWithIndex, index int) map[string]int {
+		accum[common.Bytes2Hex(next.Validator.Validator.WithdrawalCredentials[12:])] = 1
+		return accum
+	}, map[string]int{}))
+
+	fmt.Printf("Querying %d beacon-chain withdrawal addresses to see if they may be eigenpods\n", len(interestingWithdrawalAddresses))
+
+	podOwners, err := multicall.DoManyAllowFailures[common.Address](args.Mc, lo.Map(interestingWithdrawalAddresses, func(address string, index int) *multicall.MultiCallMetaData[common.Address] {
```

Comment:
failures only include reverts, right? not rpc fails

Reply:
i think it's both, that's a good point though. practically for the sensitivity of testnet and mainnet we can re-run the script and validate that 0 pods have issues